python/tests/parsers/test_parser_class.py

```diff
@@ -52,32 +52,36 @@ def new_parser(filing_detail_text_13f, primary_doc_xml_text, infotable_xml_text)
         return Parser('')
 
 
-PARSER = new_parser(filing_detail_text_13f(), primary_doc_xml_text(), infotable_xml_text())
+@pytest.fixture()
+def parser():
```

Comment:
since this is an expensive call, consider setting the scope so it isn't recreated for each test.